src/network/logout.test.ts

```diff
@@ -3,29 +3,34 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
+import { api } from '@zextras/carbonio-ui-soap-lib';
 import { http, HttpResponse } from 'msw';
 
 import { logout } from './logout';
 import * as utils from './utils';
 import server from '../mocks/server';
 import { useLoginConfigStore } from '../store/login/store';
 import { controlConsoleError } from '../tests/utils';
-import type { ErrorSoapResponse } from '../types/network';
+
```

Comment:
This way to fix tests invalidate the tests meaning.
I think is better to not mock the api.endSession but continue to mock with msw